cli/app/app.go

```diff
@@ -203,6 +205,7 @@ func Up(c *cli.Context) {
 
 // Down deletes all deployment, svc.
 func Down(c *cli.Context) {
+	CheckForUnknownArgs(c)
```

Comment:
Did you want to update this function and actually use `ValidateFlags` as mentioned by @kadel